src/Calculator/Views/GraphingCalculator/GraphingSettings.xaml.cpp

```diff
@@ -37,11 +37,11 @@ Style ^ GraphingSettings::SelectTextBoxStyle(bool incorrectRange, bool error)
 {
     if (incorrectRange || error)
     {
-        return static_cast<::Style ^>(this->Resources->Lookup(L"ErrorTextBoxStyle"));
+        return static_cast<::Style ^>(Application::Current->Resources->Lookup(L"VariableTextErrorBoxStyle"));
     }
     else
     {
-        return nullptr;
+        return static_cast<::Style ^>(Application::Current->Resources->Lookup(L"ThemedVariableTextBoxStyle"));
```

Comment:
The textbox won't be updated when the user will modify the app theme. 

We should try something like:
`<Style x:Key="GraphSettingsVariableTextBoxStyle" BasedOn="{ThemeResource ThemedVariableTextBoxStyle}" />` (in GraphingSettings.xaml)
and write
`return static_cast<::Style ^>(this->Resources->Lookup(L"GraphSettingsVariableTextBoxStyle"));`

Reply:
Hmm, are you sure? It seems to be working for me

Reply:
it works because GraphSettings is automatically closed when the window loses the focus. But if we want this popup to persist or when the theme is changed automatically (for example one of the many apps switching your theme automatically at night), the style used here won't be updated.

Reply:
I made the change, but I couldn't use ThemeResource because I got this error:
Style BasedOn property must be a Style, not 'ThemeResourceExtension' object

Still works for me, but unsure if the automatic theme changing will function correctly.